src/views/Event/index.tsx

```diff
@@ -0,0 +1,171 @@
+
+const meta: ViewMetadata = {
+  name: 'Event',
+  path: `${import.meta.env.BASE_URL || ''}/event`,
+  widths: {
+    sm: 12,
+    md: 12,
+    lg: 6,
+    xl: 6,
+    '2xl': 6,
+    hd: 6,
+    fhd: 4,
+    qhd: 3,
+    uhd: 2
+  }
+}
+
+export const Event = (props: ViewProps & { document?: Y.Doc }): JSX.Element => {
+  const query = useQuery()
+  const documentId = props.id || query.id
+
+
+  return (
+    <>
+      {documentId
+        ? <AwarenessDocument documentId={documentId} document={props.document}>
+          <EventViewContent {...props} documentId={documentId} />
+        </AwarenessDocument>
+        : <></>
+      }
+    </>
+  )
+}
+
+const EventViewContent = (props: ViewProps & { documentId: string }): JSX.Element | undefined => {
+  const { provider } = useCollaboration()
+  const { data, status } = useSession()
+  const [documentStatus, setDocumentStatus] = useDocumentStatus(props.documentId)
+
+  const [eventTitle] = useYValue<string | undefined>('root.title')
+
+  const viewVariants = cva('flex flex-col', {
+    variants: {
+      asCreateDialog: {
+        false: 'h-screen',
+        true: 'overflow-hidden'
+      }
+    }
+  })
+
+  const sectionVariants = cva('overscroll-auto @5xl:w-[1024px] space-y-5', {
+    variants: {
+      asCreateDialog: {
+        false: 'p-8',
+        true: 'p-6'
+      }
+    }
+  })
+
+  return (
+    <div className={cn(viewVariants({ asCreateDialog: !!props.asCreateDialog, className: props?.className }))}>
+      <div className='grow-0'>
+        <ViewHeader.Root>
+          {!props.asCreateDialog &&
+            <ViewHeader.Title title='Händelse' icon={Ticket} iconColor='#DAC9F2' />
+          }
+
+          <ViewHeader.Content>
+            <div className='flex w-full h-full items-center space-x-2'>
+              <DocumentStatus status={documentStatus} setStatus={setDocumentStatus} />
+              <VisibilityStatus />
+              <Newsvalue />
+            </div>
+          </ViewHeader.Content>
+
+          <ViewHeader.Action onDialogClose={props.onDialogClose}>
+            {!props.asCreateDialog && !!props.documentId &&
+              <ViewHeader.RemoteUsers documentId={props.documentId} />
+            }
+          </ViewHeader.Action>
+        </ViewHeader.Root>
+      </div>
+
+      <ScrollArea className='grid @5xl:place-content-center'>
+        <section className={cn(sectionVariants({ asCreateDialog: !!props?.asCreateDialog }))}>
+          <div className='flex flex-col gap-2 pl-0.5'>
+            <div className='flex space-x-2 items-start'>
+              <Title
+                autoFocus={props.asCreateDialog}
+                placeholder='Händelserubrik'
+              />
+            </div>
+
+            <Description role='public' />
+            <Registration />
+
+            <p>Datetime TODO</p>
+          </div>
+
+          <div className='flex flex-col space-y-2 w-fit'>
+            <div className='grid grid-cols-2 gap-2 @xs:flex'>
```

Comment:
```suggestion
            <div className='flex flex-wrap gap-2'>
```

Reply:
Does that work for you? Cause that was my first try, but it didn't work as expected :thinking: 
I'll try again.